code/__defines/mobs.dm

```diff
@@ -166,3 +166,12 @@
 
 #define RESPAWN_ANIMAL 3000
 #define RESPAWN_MINISYNTH 6000
+
+//Flags for the eat_types variable, a bitfield of what can or can't be eaten
+//Note that any given mob can be more than one type
+#define TYPE_ORGANIC	1//Almost any creature under /mob/living/carbon and most simple animals
+#define	TYPE_SYNTHETIC	2//Everything under /mob/living/silicon, plus IPCs, viscerators
+#define TYPE_HUMANOID	4//Humans, skrell, unathi, tajara, vaurca, diona, IPC, vox
+#define TYPE_WIERD		8//Slimes, constructs, demons, and other creatures of a magical or bluespace nature.
+
```

Comment:
Moved these out of mob helpers up to here. I could swear ive done this before, i think baymerge reverted it